jeeves/report.py

```diff
@@ -12,6 +12,36 @@
 from jeeves.blockers import get_bugs_dict, get_bugs_set, get_tickets_dict, get_tickets_set, get_other_blockers
 
 
+def generate_common_stats_information(num_success, num_unstable, num_failure, num_covered, num_aborted, num_missing, num_error, num_jobs):
+	summary = {}
+	summary['total_jobs'] = "Total number of jobs: {}".format(num_jobs)
+	summary['total_success'] = "SUCCESS:  {}/{} = {}%".format(num_success, num_jobs, percent(num_success, num_jobs))
+	summary['total_unstable'] = "UNSTABLE: {}/{} = {}%".format(num_unstable, num_jobs, percent(num_unstable, num_jobs))
+	summary['total_failure'] = "FAILURE:  {}/{} = {}%".format(num_failure, num_jobs, percent(num_failure, num_jobs))
+	if num_covered > 0:
+		summary['total_coverage'] = "Blocker Coverage:  {}/{} = {}%".format(num_covered, num_jobs - num_success, percent(num_covered, num_jobs - num_success))
+	else:
+		summary['total_coverage'] = False
+	# include abort report if needed
+	if num_aborted > 0:
+		summary['total_aborted'] = "ABORTED:  {}/{} = {}%".format(num_aborted, num_jobs, percent(num_aborted, num_jobs))
+	else:
+		summary['total_aborted'] = False
+
+	# include missing report if needed
+	if num_missing > 0:
+		summary['total_missing'] = "NO_KNOWN_BUILDS:  {}/{} = {}%".format(num_missing, num_jobs, percent(num_missing, num_jobs))
+	else:
+		summary['total_missing'] = False
+
+	# include error report if needed
+	if num_error > 0:
+		summary['total_error'] = "ERROR:  {}/{} = {}%".format(num_error, num_jobs, percent(num_error, num_jobs))
+	else:
+		summary['total_error'] = False
+	return summary
+
+
```

Comment:
Move to jeeves.common - maybe we can call this `generate_summary`?

Reply:
Done